evalml/utils/base_meta.py

```diff
@@ -30,5 +30,6 @@ def __new__(cls, name, bases, dct):
                 property_orig = dct[attribute]
                 dct[attribute] = property(cls.check_for_fit(property_orig.__get__),
                                           property_orig.__set__,
-                                          property_orig.__delattr__)
+                                          property_orig.__delattr__,
+                                          property_orig.__doc__)
```

Comment:
Yep! So, for any pipeline or component property attribute, we now correctly pass along the `__doc__` content for sphinx. Right on.